benchmark/add/results

```diff
@@ -1,13 +1,13 @@
 
 Testing interpreted
-total 3459.1666850000 ms
-mean 345.9166685000 ns
+total 4813.7201410000 ms
+mean 481.3720141000 ns
 
 Testing native
-total 217.1513420000 ms
-mean 21.7151342000 ns
+total 255.1195230000 ms
+mean 25.5119523000 ns
 
 Interations: 10000000
-Date: 2017-12-29
+Date: 2018-1-6
 V8 version: 6.2.414.46-node.15
-Interpreter version: 0.0.14
+Interpreter version: 0.0.15
```

Comment:
We can totally merge this file, but I should probably run it from my machine to keep it consistent, your native result seems a bit slower.